docs/tutorials/01-introduction.md

```diff
@@ -90,11 +90,11 @@ If you don't see the event-source and sensor pod in `argo-events` namespace,
   4. Look for any error in event-source or sensor pod.
   5. Inspect the event-source,
 
-        kubectl -n argo-events event-source-object-name -o yaml
+        kubectl -n argo-events get pod event-source-object-name -o yaml
 
      Inspect the sensor,
 
-        kubectl -n argo-events sensor-object-name -o yaml
+        kubectl -n argo-events get pod sensor-object-name -o yaml
```

Comment:
Should be:
```
kubectl -n argo-events get sensor sensor-object-name -o yaml
```

Reply:
This paragraph needs to be updated, some of them do not apply any more:

(2) can be removed, we don't use configmap any more;
(4) should also be removed, because the subject is `If you don't see the event-source and sensor pod in xxx`
(5) should be moved to (1).